src/routes/submit-route/submitted-slops/submitted-slops-route.jsx

```diff
@@ -0,0 +1,40 @@
+import { useContext } from "react";
+import { Link } from "react-router-dom";
+import { Footer, Header } from "../../../components";
+import { CurrentUserContext } from "../../../store/current-user-context";
```

Comment:
you will need to change this import to `import { useCurrentUser } from '../../../hooks';`